### PR TITLE
{bio}[foss/2020a] bcbio-gff v0.6.6 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by manifestoso**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+bcbiogffroot = 'lib/python%(pyshortver)s/site-packages'
```
Comment (verdurin) [inline easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb:32]: @robqiao please use use the `local_` prefix for these. See https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html for details - thanks.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (robqiao) [reply easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb]: Thx @verdurin, local `--check-contrib` didn't pick up this one.

Comment (verdurin) [reply easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.6-foss-2020a-Python-3.8.2.eb]: Thanks.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/e41c98943a417f2a1e77d3596252ea0a for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/b528e69ce41d7584429a5c38c4c62691 for a full test report.

**Comment by verdurin** [comment]
@manifestoso the build failure is:
```
== 2021-11-11 12:49:30,994 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Sanity check failed: command "/eb/maint/software/Python/3.8.2-GCCcore-9.3.0/bin/python -c "import BCBio.GFF"" failed; output:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/eb/maint/software/bcbio-gff/0.6.6-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/BCBio/GFF/__init__.py", line 3, in <module>
    from BCBio.GFF.GFFParser import GFFParser, DiscoGFFParser, GFFExaminer, parse, parse_simple
  File "/eb/maint/software/bcbio-gff/0.6.6-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/BCBio/GFF/GFFParser.py", line 32, in <module>
    from Bio.Seq import UnknownSeq
ModuleNotFoundError: No module named 'Bio' (at easybuild/framework/easyblock.py:3311 in _sanity_check_step)
```

Missing dep on `BioPython`?

**Comment by manifestoso** [comment]
Test report by @manifestoso
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
hpc-login01 - Linux RHEL 7.5, x86_64, AMD EPYC 7551 32-Core Processor, Python 3.6.10
See https://gist.github.com/41c44d7ad648bfec16710f48afb9c053 for a full test report.

**Comment by manifestoso** [comment]
Thx @verdurin, LGTM now.

**Comment by manifestoso** [comment]
> 
> 
> @manifestoso the build failure is:
> 
> ```
> == 2021-11-11 12:49:30,994 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Sanity check failed: command "/eb/maint/software/Python/3.8.2-GCCcore-9.3.0/bin/python -c "import BCBio.GFF"" failed; output:
> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
>   File "/eb/maint/software/bcbio-gff/0.6.6-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/BCBio/GFF/__init__.py", line 3, in <module>
>     from BCBio.GFF.GFFParser import GFFParser, DiscoGFFParser, GFFExaminer, parse, parse_simple
>   File "/eb/maint/software/bcbio-gff/0.6.6-foss-2020a-Python-3.8.2/lib/python3.8/site-packages/BCBio/GFF/GFFParser.py", line 32, in <module>
>     from Bio.Seq import UnknownSeq
> ModuleNotFoundError: No module named 'Bio' (at easybuild/framework/easyblock.py:3311 in _sanity_check_step)
> ```
> 
> Missing dep on `BioPython`?

Thx @verdurin, LGTM now.

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by manifestoso** [comment]
@verdurin @migueldiascosta Can we merge this PR, as multiple PR deps on this one. Thx

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/50ac962df39791cfb5a5e11d8ef91963 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @manifestoso!

**Pull request merged**